src/shared/seccomp-util.h

```diff
@@ -115,3 +118,27 @@ DEFINE_TRIVIAL_CLEANUP_FUNC(scmp_filter_ctx, seccomp_release);
 int parse_syscall_archs(char **l, Set **ret_archs);
 
 uint32_t scmp_act_kill_process(void);
+
+/* This is a special value to be used where syscall filters otherwise expect errno numbers, will be
+   replaced with real seccomp action. */
+typedef enum SeccompErrorNumber {
+        SECCOMP_ERROR_NUMBER_KILL = INT_MAX - 1,
+} SeccompErrorNumber;
```

Comment:
you never use the type you define here, just the enum value. hence, please make this anonymous, i.e.: 

```
enum {
        SECCOMP_ERRNO_KILL = INT_MAX - 1,
};
assert_cc(SECCOMP_ERRNO_KILL > ERRNO_MAX); /* Safety check so that we do not step into Linux errno territory. */
```